test/clj/rems/api/services/test_catalogue.clj

```diff
@@ -118,8 +118,9 @@
       (archive-resource! true)
       (archive-license! true)
       (let [errors (:errors (archive-catalogue-item! false))]
+        ;; TODO indirect catalogue item -> resource -> license dep not tracked right now
```

Comment:
I guess it is quite easy to just expand the transitive dependencies? Instead of using the `(dependencies)` as a direct map you could calculate them on the fly in a helper function or just expand them directly to the cached map?

Reply:
Yes, but I didn't want to change the behaviour (yet)

Reply:
We can definitely do that in a follow-up PR.